### DISCUSSION TITLE
Gizmo user port not connecting

### DISCUSSION THREAD

**Discussion by VandalRobotics** [gizmo-platform/gizmo#33] [Q&A]
Whenever we try and connect it through the user port nothing shows up on our computers. The system port works fine but the user port does not work. I cannot install any firmware that I need on the user port or put any code on it to.

**Comment by Vaelatern** [comment]
Have you followed the guide for flashing including holding down the appropriate button while powering up the gizmo or student processor?

**Reply by VandalRobotics** [reply]
I'm pretty sure but I will try again whenever I get back to work on it again.

**Reply by VandalRobotics** [reply]
I just tried and it did not work

**Reply by Vaelatern** [reply]
Reach out to your local support channels to confirm if you need a replacement.
